src/nvim/event/process.c

```diff
@@ -212,11 +212,13 @@ int process_wait(Process *proc, int ms, MultiQueue *events)
 }
 
 /// Ask a process to terminate and eventually kill if it doesn't respond
-void process_stop(Process *proc) FUNC_ATTR_NONNULL_ALL
+/// @return false when the proc has already exited or stopped
```

Comment:
as justinmk said this change should be redundant now that we use `find_job` with false (in case it was not clear: I was only against adding a new special -1 value, not anything else that was said).